src/core_test/mpas_test_core_streams.F

```diff
@@ -391,6 +391,205 @@ subroutine test_core_streams_test(domain, threadErrs, ierr)
+         ierr = ierr + abs(err)
+      end if
+
+   end subroutine test_remove_alarm
+
+   !***********************************************************************
+   !  Subroutine test_remove_output_alarm
+   !
+   !> \brief   Wrapper for testing the removal of an output alarm from a stream.
+   !>
+   !> \details This subroutine calls the `test_remove_alarm` subroutine,
+   !>          passing the `MPAS_STREAM_OUTPUT` as the stream type to test
+   !>          the functionality of removing an output alarm from a stream.
+   !
+   subroutine test_remove_output_alarm(domain, ierr)
+      use mpas_stream_manager
+      implicit none
+      type(domain_type), intent(inout) :: domain
+      integer, intent(out) :: ierr
+
+      call test_remove_alarm(domain, ierr, MPAS_STREAM_OUTPUT)
+   end subroutine test_remove_output_alarm
+
+   !***********************************************************************
+   !  Subroutine test_remove_input_alarm
+   !
+   !> \brief   Wrapper for testing the removal of an input alarm from a stream.
+   !>
+   !> \details This subroutine calls the `test_remove_alarm` subroutine,
+   !>          passing the `MPAS_STREAM_INPUT` as the stream type to test
+   !>          the functionality of removing an input alarm from a stream.
+   !
+   subroutine test_remove_input_alarm(domain, ierr)
+      use mpas_stream_manager
+      implicit none
+      type(domain_type), intent(inout) :: domain
+      integer, intent(out) :: ierr
+
+      call test_remove_alarm(domain, ierr, MPAS_STREAM_INPUT)
+   end subroutine test_remove_input_alarm
+
+   !***********************************************************************
+   !  Subroutine test_core_streams_test
+   !
+   !> \brief   Core test suite for stream I/O and alarm management in MPAS.
+   !>
+   !> \details This subroutine tests the functionality of reading/writing
+   !>          real-valued streams and managing alarms in streams. It calls
+   !>          individual tests for stream I/O operations and for adding/removing
+   !>          alarms from input and output streams. The results of each test
+   !>          are logged with a success or failure message.
+   !>
+   !> \param domain      The domain object that contains the stream manager
+   !>                    and clock.
+   !> \param threadErrs  An array to store any errors encountered during
+   !>                    the test.
+   !> \param ierr        The error code that indicates the result of the test.
+   !
+   subroutine test_core_streams_test(domain, threadErrs, ierr)
+
+      use mpas_log
+
+      implicit none
+
+      type (domain_type), intent(inout) :: domain
+      integer, dimension(:), intent(out) :: threadErrs
+      integer, intent(out) :: ierr
+
+      integer, dimension(3) :: test_status
+
+      ierr = 0
+      test_status = 0
+
+      call mpas_log_write('Testing reading/writing real-valued streams')
+
+      call test_read_write_real_streams(domain, threadErrs, test_status(1))
+
```

Comment:
Here and on line 567, removing the blank lines would give more consistent whitespace with the blocks of code for alarm removal tests below.